cli/script/command-executor.ts

```diff
@@ -1223,6 +1364,43 @@ function requestAccessToken(): Promise<string> {
     });
 }
 
+export var runReactNativeBundleCommand = (entryFile: string, outputFolder: string, platform: string, sourcemapOutput: string): Promise<void> => {
+    var reactNativeBundleCommandArgs = [
```

Comment:
Ah, I forgot that the local CLI does a bunch of work (the relationship between the global CLI, local CLI and packager is pretty confusing), so you're totally right that we shouldn't consume the packager for this.

If we consumed the local CLI directly would we need to duplicate a bunch of work in the global CLI? If we absolutely have to use the global CLI, then that's cool, I'm just curious if that is the case.